src/markdownPreview/edit.html

```diff
@@ -1,6 +1,13 @@
 <form role="form">
   <div class="form-group">
-    <p><label for="repository">Repository</label>
+    <p>
+      <label for="repository">Repository</label>
+      <span class="grey glyphicon glyphicon-info-sign ng-scope" tabindex="0"
```

Comment:
Why using 2 variables here? `tooltipIsOpenRepo` and `repositoryTooltipIsOpen`
Some attributes are defined 2 times: e.g. `tooltip-is-open`